getting-started/faqs.md

```diff
@@ -4,220 +4,197 @@
 
 ### About CFT
 
-**1. What is Cloud File Transfer (CFT)?**
+**What is Cloud File Transfer (CFT)?**
 
-A centralized, fully managed file transfer service for the whole of government (WOG). CFT is designed to help you send and receive large files securely across zones. The sender and receiver can be an application or an end-user. Use CFT to transfer files between your intranet and internet zones or exchange files with your business partners and vendors as well as other agencies. 
+A centralized, fully managed file transfer service for the whole of government (WOG).
 
 CFT is a part of the SG Tech Stack (SGTS) and is hosted on the Government Commercial Cloud (GCC).
 
-**Key Features:**
+Core capabilities:
 
-**Compliant file transfers**
+- Compliant cross-zone file transfers according to IM8
+- Malware scanning with Content Disarm and Reconstruct (CDR)
+- File Encryption for data integrity and security
+- Application Programming Interface (API) access for web and mobile applications
+- Event-driven with file download notifications
 
-CFT complies with Instruction Manual (IM8) Application Infrastructure Architecture Standard (AIAS) policy guidelines.
+**What are the benefits of using CFT?**
 
-**Malware scanning and Content Disarm and Reconstruct (CDR)** 
+CFT can benefit agencies who require to:
 
-Files are scanned for malware during transfer and malicious content is removed by the CDR functionality. Latest Antivirus (AV) updates are automatically applied to your files when they are being scanned.
+- Transfer large files across zones
+- Move files securely
+- Automatically scale based on their transfer needs
+- Customize workflows
+- Audit and monitor their file transfer activity
+- Accelerate digitization of government services
 
-**Large file transfers**
+**Who are the end users?**
 
-Up to 1GB per transaction.
+CFT is available for government agencies, their business partners, and vendors.
 
-**Application Programming Interface (API) access for web and mobile applications** 
+**What are the different integration options?**
 
-Your applications can communicate with CFT through its APIs enabling you to access CFT to send, scan and receive files.
+CFT will be available for the following integration options:
 
-**Event-driven** 
+1. CFT for Backend Systems - For file transfers between systems.
+2. CFT for Web and Mobile Applications - For file transfers between an end-user and a system.
 
-Get notified when files are available for download either by setting up a webhook or by registering your email for updates.
```

Comment:
is this supposed to be displayed? dont see it in my md viewer. 

if yes, the intent is to show the reader our added value that we will notify them whenever there are files available for download?